Formula/metricbeat.rb

```diff
@@ -1,9 +1,11 @@
 class Metricbeat < Formula
   desc "Collect metrics from your systems and services"
-  homepage "https://www.elastic.co/products/beats/metricbeat"
+  homepage "https://www.elastic.co/beats/metricbeat"
   url "https://github.com/elastic/beats.git",
-      tag:      "v7.8.0",
-      revision: "f79387d32717d79f689d94fda1ec80b2cf285d30"
+      {
+        tag:      "v7.8.1",
+        revision: "94f7632be5d56a7928595da79f4b829ffe123744",
+      }
```

Comment:
This technically passes audit but please don't do this.

Reply:
@SeekingMeaning I am not a Ruby guy (yet). What is the right way to do it?

Reply:
How it was before is fine; Ruby automatically converts it to a hash so you don't need `{` and `}`

Reply:
@SeekingMeaning I am pretty sure yesterday it didn't pass the CI tests because it was complaining that I didn't use the new Ruby 1.9 hash syntax but today it works. So I changed it back